values.yaml

```
@@ -655,39 +670,39 @@ ruler:
   # -- allow configuring rules via configmap. ref: https://cortexproject.github.io/cortex-helm-chart/guides/configure_rules_via_configmap.html
   directories: {}
 
-  # -- Sidecars that collect the configmaps with specified label and stores the included files them into the respective folders
   sidecar:
+    # -- Enable sidecar that collect the configmaps with specified label and stores the included files them into the respective folders
+    enabled: false
     image:
-      repository: quay.io/kiwigrid/k8s-sidecar
-      tag: 1.10.7
+      repository: omegavveapon/kopf-k8s-sidecar
+      tag: 1.4.0
       sha: ""
     imagePullPolicy: IfNotPresent
     resources: {}
-    #   limits:
-    #     cpu: 100m
-    #     memory: 100Mi
-    #   requests:
-    #     cpu: 50m
-    #     memory: 50Mi
-    # skipTlsVerify Set to true to skip tls verification for kube api calls
-    # skipTlsVerify: true
+    # -- Set to true to skip tls verification for kube api calls
+    skipTlsVerify: false
+    # -- A value of true will produce unique filenames to avoid issues when duplicate data keys exist between ConfigMaps
+    # and/or Secrets within the same or multiple Namespaces.
     enableUniqueFilenames: false
-    enabled: false
     # -- label that the configmaps with rules are marked with
     label: cortex_rules
+    # -- Determines how kopf-k8s-sidecar will run. If WATCH it will run like like a normal operator forever.
+    # If LIST it will gather the matching configmaps and secrets currently present,
+    # write those files to the destination directory and die
     watchMethod: null
```

```suggestion
    watchMethod: ""
```

I don't think we use `null` anywhere else. maybe we should change that for consistency